Blogger.API/Api/Blogs/BlogController.cs

```diff
@@ -81,5 +81,12 @@ public async Task<ActionResult> Update([FromBody] UpdateBlogRequest updateBlogRe
 
             return Ok();
         }
+
+        [HttpDelete("{id}")]
+        public async Task<ActionResult> SoftDeleteAsync(Guid id)
```

Comment:
Rename this to DeleteAsync. "Soft" is a type of delete and is an implementation detail of our core